lib/dgram.js

```diff
@@ -146,7 +146,6 @@ Socket.prototype.bind = function(port /*, address, callback*/) {
   if (typeof arguments[arguments.length - 1] === 'function')
     self.once('listening', arguments[arguments.length - 1]);
 
-  const UDP = process.binding('udp_wrap').UDP;
```

Comment:
This was a redeclaration of [line 8](https://github.com/iojs/io.js/blob/v1.x/lib/dgram.js#L8)